R/github-actions.R

```diff
@@ -240,7 +233,7 @@ use_tidy_github_actions <- function(ref = NULL) {
     }
   }
 
-  invisible(full_status && pr_status && pkgdown_status && test_coverage_status)
```

Comment:
We spent a lot of code collecting information that's never used anywhere. I think code is clearer without it.